.eslintrc.js

```diff
@@ -10,6 +10,12 @@ module.exports = {
     sourceType: 'module',
   },
   rules: {},
+  settings: {
+    react: {
+      pragma: 'preact',
```

Comment:
Shouldn't this be
```suggestion
      pragma: 'h',
```
?

Reply:
Seems no difference to the result since we don't import `h` in source code, but I guess `h` is the right one.